src/library_manager/llext_manager.c

```diff
@@ -35,6 +36,7 @@
 #include <rimage/sof/user/manifest.h>
 #include <module/module/api_ver.h>
 
+#include <adsp_memory_regions.h>
```

Comment:
I think the west update needs to come before this, or bisect is broken.

Reply:
so it does... 
<img width="682" height="151" alt="image" src="https://github.com/user-attachments/assets/bd5ba39c-38d1-4afe-b3f3-2f632e5ab8de" />